package.json

```
@@ -86,5 +88,13 @@
     "picomatch": {
       "optional": true
     }
+  },
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
```

```suggestion
  "module": "./dist/index.mjs",
  "exports": {
    ".": {
      "import": "./dist/index.mjs",
```
small thing missed when enabling `fixedExtension`